packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [38.1.0]
+
+### Changed
+
+- Parallelization of detected tokens with balance ([#4697](https://github.com/MetaMask/core/pull/4697))
+- Bump accounts related packages ([#4713](https://github.com/MetaMask/core/pull/4713)), ([#4728](https://github.com/MetaMask/core/pull/4728))
+  - Those packages are now built slightly differently and are part of the [accounts monorepo](https://github.com/MetaMask/accounts).
+  - Bump `@metamask/keyring-api` from `^8.1.0` to `^8.1.4`
```

Comment:
```suggestion
```

Reply:
Yes true, but I wanted to mark those since the introduction of our new monorepo on the account side. We have done some basic testing on the extension side (using our e2e and some manual testing with preview builds) but in case there's any regression (there should not) I thought it would be great to have this being commented somewhere in the CHANGELOG.

If you think that's "too much", I'm also ok removing those!

Reply:
Sounds perfectly reasonable to keep them. Just wanted to be sure their inclusion was intentional.